backend/tests/services/knowledge/group_kb/test_group_kb_inheritance.py

```
@@ -0,0 +1,128 @@
+            data=kb_data,
+        )
+
+        # Verify owner has manage permission
+        kb = test_db.query(Kind).filter(Kind.id == kb_id).first()
+        level = _get_user_kb_permission_level(test_db, kb, test_group_owner.id)
+        assert level == "manage"
+
+    def test_developer_inherits_to_subgroup(
+        self,
+        test_db: Session,
+        test_group,
```

Ruff (0.15.2)

[warning] Unused method argument: `test_group`

(ARG002)